Applications/Spire/Include/Spire/KeyBindings/CompositeKeyBindingsModel.hpp

```diff
@@ -0,0 +1,48 @@
+#ifndef SPIRE_COMPOSITE_KEY_BINDINGS_MODEL_HPP
+#define SPIRE_COMPOSITE_KEY_BINDINGS_MODEL_HPP
+#include "Spire/KeyBindings/KeyBindingsModel.hpp"
+
```

Comment:
Include the ```KeyBindings.hpp``` header, which in turn provides forward declarations for all key binding related types.